pkg/insights/resyncer.go

```diff
@@ -557,32 +557,34 @@ func getOrDefault(version string) string {
 	return version
 }
 
-func (r *resyncer) needResyncServiceInsight(mesh string) bool {
-	info, exist := r.getInfo(ServiceInsightKey(mesh))
+func (r *resyncer) needResyncServiceInsight(ctx context.Context, mesh string) bool {
+	info, exist := r.getInfo(ctx, ServiceInsightKey(mesh))
 	return !exist || core.Now().Sub(info.lastSync) > r.minResyncTimeout
 }
 
-func (r *resyncer) needResyncMeshInsight(mesh string) bool {
-	info, exist := r.getInfo(MeshInsightKey(mesh))
+func (r *resyncer) needResyncMeshInsight(ctx context.Context, mesh string) bool {
+	info, exist := r.getInfo(ctx, MeshInsightKey(mesh))
 	return !exist || core.Now().Sub(info.lastSync) > r.minResyncTimeout
 }
 
-func (r *resyncer) getInfo(key model.ResourceKey) (syncInfo, bool) {
+func (r *resyncer) getInfo(ctx context.Context, key model.ResourceKey) (syncInfo, bool) {
 	r.infosMux.RLock()
 	defer r.infosMux.RUnlock()
-	info, ok := r.infos[key]
+	tenantID, _ := multitenant.TenantFromCtx(ctx)
```

Comment:
why are we ignoring the second value? do we even need this if we're ignoring it every time?

Reply:
Yes, otherwise it does not compile. We ignore this because it does not matter if the value is here or not. The logic is the same